api/lib/searchOperators.js

```diff
@@ -69,6 +92,38 @@ const getQueryConditionsAndParams = (
   };
 };
 
+const hasBindiCharacter = charCode => {
+  if (bindiCharacters[charCode]) {
+    return bindiCharacters[charCode];
+  }
+  return false;
+};
+
+const generateBindiQuery = (charCodeQuery, charCodeQueryWildcard, result) => {
```

Comment:
Because it won't always generate a bindi query, i think this should be named as preprocessQuery (or something similar, maybe much simpler). 

This will allow us to add more preprocessing in future if needed.

Reply:
It would always generate a bindi query for now, we won't be using it for something else.